src-python/grpc-stub/server.py

```diff
@@ -47,6 +47,16 @@ def SetLoginDetails(self, request, context):
         return noviflow_pb2.CliReply(reply_status=0)
 
     def SetConfigLogicalPort(self, request, context):
+        if request.logicalportno not in range(100, 63488):
```

Comment:
I think `100 <= request.logicalportno <= 63488:` is better because `range` generated an iterator and check elements in range form 100 to 63488. Instead `a <= x <= b` uses 2 comparisons. So it is faster.